blocks/rich-text/index.js

```diff
@@ -529,6 +532,10 @@ export default class RichText extends Component {
 		const dom = this.editor.dom;
 		const rootNode = this.editor.getBody();
 
+		if ( isAccess( event ) ) {
+			this.fireChange();
```

Comment:
Haven't followed the logic, but in what way is an access keydown a change?

Reply:
Otherwise the state would not be up to date at the time of the command. This Is an ugly fix. There are PRs in the works that sync `RichText` state continuously though, and then this would no longer be needed.